.github/workflows/docker_build.yml

```diff
@@ -97,4 +118,24 @@ jobs:
             NODE_ENV=${{ inputs.node_env }}
             NPM_TOKEN=${{ secrets.NPM_TOKEN }}
             OMNI_COMPONENT_COMMIT=${{ github.sha }}
-            OMNI_COMPONENT_VERSION=${{ steps.meta.outputs.version }}
+            OMNI_COMPONENT_VERSION=${{ steps.meta.outputs.version }}       
+      
+
+          
+      - name: save docker image to tar file
+        if: env.MINIO_CN_ENDPOINTS != null && env.MINIO_CN_ACCESS_KEY_ID != null && env.MINIO_CN_SECRET_ACCESS_KEY != null
+        run: docker save --output ./${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }} ${{ steps.meta.outputs.tags }}
+
+      - name: upload docker image to private minio storage
+        if: env.MINIO_CN_ENDPOINTS != null && env.MINIO_CN_ACCESS_KEY_ID != null && env.MINIO_CN_SECRET_ACCESS_KEY != null
+        run: |   
+          export AWS_ACCESS_KEY_ID=${{ secrets.MINIO_CN_ACCESS_KEY_ID }}
+          export AWS_SECRET_ACCESS_KEY=${{ secrets.MINIO_CN_SECRET_ACCESS_KEY }}
+          aws --endpoint-url ${{ secrets.MINIO_CN_ENDPOINT_URL }} s3 cp  ./${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}  s3://docker-img-artifacts/${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }} 
+
+
+    outputs:
+      imageTag: ${{ steps.meta.outputs.tags }}
+      imageTarName: ${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}
```

Comment:
same, update the name to `image_tag` and `image_tar_name`. 

the  `image-tag` or `image-tar-name` are also a good name but you need to make sure all vars use the same style. e.g. `docker_file` to `docker-file`